import-export-cli/integration/testutils/operationPolicy_testUtils.go

```diff
@@ -0,0 +1,552 @@
+
+	assert.NotNil(t, err, "Error while importing the API Policy")
+	assert.Contains(t, importOutput, "500", "Error importing API Policy")
+}
+
+// ValidateMalformedAPIPolicyExportImport : Validates Exporting API Policy from source env and Importing to destination env
+func ValidateAPIPolicyImportFailureWhenPolicyExisted(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[PolicyNameKey])
+	policyVersion := fmt.Sprintf("%v", args.Policy[PolicyVersionKey])
+
+	args.SrcAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	exportedOutput, _ := exportAPIPolicy(t, policyName, args.SrcAPIM.GetEnvName())
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+
+	fmt.Println("Exported Path: ", args.ImportFilePath)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+
+	args.SrcAPIM.DeleteAPIPolicy(fmt.Sprintf("%v", args.Policy[PolicyIdKey]), "Export")
+
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Import API Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	args.DestAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	importedOutput, err := importAPIPolicy(t, args)
+
+	policyId := args.DestAPIM.GetAPIPolicyID(t, policyName, policyVersion)
+
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully Imported API Policy.")
+
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	importedOutput, _ = importAPIPolicy(t, args)
+
+	// assert.NotNil(t, err, "Error importing API Policy")
+	assert.Contains(t, importedOutput, "Error importing API Policy")
+
+	cleanUpImportExportPolicies(t, args, policyId)
+
+}
+
+// Adds a new API Policy to an env
+func AddNewAPIPolicy(t *testing.T, client *apim.Client, username, password, pathToSpec, pathToSynapse string) interface{} {
+	client.Login(username, password)
+	pathToPolicySpecFile, _ := filepath.Abs(pathToSpec)
+	pathToSynapseDefFile, _ := filepath.Abs(pathToSynapse)
+	policyContent := readAPIPolicyDefinition(t, pathToPolicySpecFile)
+	policySpecFileData, err := json.Marshal(policyContent)
+
+	// synapseDefFileData, err := ioutil.ReadFile(pathToSynapseDefFile)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	createdPolicy := client.AddAPIpolicy(t, policySpecFileData, pathToSynapseDefFile)
+	return createdPolicy
+}
+
+// Exports API Policy from an env
+func exportAPIPolicy(t *testing.T, name, env string) (string, error) {
+	output, err := base.Execute(t, "export", "policy", "api", "-n", name, "-e", env, "-k", "--verbose")
```

Comment:
Need to add a t.Cleanup function here. Refer [1].

[1] https://github.com/wso2/product-apim-tooling/blob/7a352c13ca647dddda3bf2004cf9f6d9be95e5ff/import-export-cli/integration/testutils/policy_testUtils.go#L271